### PR TITLE
Support `/mode` for Schema Registry 

### PR DESCRIPTION
**Pull request by kbatuigas**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2619
Resolves https://github.com/redpanda-data/documentation-private/issues/2526
Review deadline: 30 July 2024

Related https://github.com/redpanda-data/docs/pull/646 (update API reference)

## Page previews

[Use the Schema Registry API > Use READONLY mode for disaster recovery](https://deploy-preview-647--redpanda-docs-preview.netlify.app/24.2/manage/schema-reg/schema-reg-api/#use-readonly-mode-for-disaster-recovery)

## Checks

- [x] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/manage/pages/schema-reg/schema-reg-api.adoc [hunk 1]
```diff
@@ -931,6 +931,53 @@ Curl::
 --
 ====
 
+== Use READONLY mode for disaster recovery
+
+The `/mode` endpoint allows you to put the Schema Registry in read-only or read-write mode. A read-only Schema Registry does not accept direct writes. An active production cluster replicates schemas to the read-only Schema Registry to keep it in sync, for example using MirrorMaker2. In a failover due to a disaster or outage, Schema Registry can be set to read-write mode, taking over for the failed cluster and ensuring availability.
```
Comment (Feediver1) [inline modules/manage/pages/schema-reg/schema-reg-api.adoc]:
```suggestion
The `/mode` endpoint allows you to put the Schema Registry in read-only or read-write mode. A read-only Schema Registry does not accept direct writes. An active production cluster replicates schemas to the read-only Schema Registry to keep it in sync, for example using MirrorMaker2. In a failover due to a disaster or outage, you can set Schema Registry to read-write mode, taking over for the failed cluster and ensuring availability. 
```

**Review by Feediver1 — APPROVED** [review]
Thanks for the quick turnaround @kbatuigas

File: modules/manage/pages/schema-reg/schema-reg-api.adoc [hunk 2]
```diff
@@ -931,6 +931,53 @@ Curl::
 --
 ====
 
+== Use READONLY mode for disaster recovery
+
+The `/mode` endpoint allows you to put the Schema Registry in read-only or read-write mode. A read-only Schema Registry does not accept direct writes. An active production cluster replicates schemas to the read-only Schema Registry to keep it in sync, for example using MirrorMaker2. In a failover due to a disaster or outage, you can set Schema Registry to read-write mode, taking over for the failed cluster and ensuring availability.
```
Comment (mattschumpert) [inline modules/manage/pages/schema-reg/schema-reg-api.adoc]: @kbatuigas  we should say 'can replicate schemas'. THis is all DIY and not built into the product.

Comment (mattschumpert) [reply modules/manage/pages/schema-reg/schema-reg-api.adoc]: "An active production cluster can replicate schemas to the read-only Schema Registry to keep it in sync, but users in the DR site will not be able to update schemas directly, ensuring the DR cluster has an exact replica of the schemas in production.  In a failover ..."

**Review by mattschumpert — APPROVED** [review]

File: modules/manage/pages/schema-reg/schema-reg-api.adoc [hunk 3]
```diff
@@ -931,6 +931,53 @@ Curl::
 --
 ====
 
+== Use READONLY mode for disaster recovery
+
+The `/mode` endpoint allows you to put Schema Registry in read-only or read-write mode. A read-only Schema Registry does not accept direct writes. An active production cluster can replicate schemas to a read-only Schema Registry to keep it in sync, for example using MirrorMaker2. Users in the disaster recovery (DR) site cannot update schemas directly, so the DR cluster has an exact replica of the schemas in production. In a failover due to a disaster or outage, you can set Schema Registry to read-write mode, taking over for the failed cluster and ensuring availability.
```
Comment (BenPope) [inline modules/manage/pages/schema-reg/schema-reg-api.adoc]: MirrorMaker won't with with our Schema Registry. We have some tools: https://github.com/redpanda-data/schema-migration/

**Pull request merged**